packages/wonder-blocks-core/util/util.js

```diff
@@ -43,13 +43,14 @@ export function processStyleList<T: Object>(
         };
     }
 
+    // Check to see if we should inline all the styles for snapshot tests.
+    const shouldInlineStyles =
+        typeof global !== "undefined" && global.SNAPSHOT_INLINE_APHRODITE;
```

Comment:
Moved this check for two reasons:

1. To reuse it
2. To make it clear what the check is for